src/OnchainKitProvider.tsx

```diff
@@ -20,17 +21,20 @@ export function OnchainKitProvider({
   if (schemaId && !checkHashLength(schemaId, 64)) {
     throw Error('EAS schemaId must be 64 characters prefixed with "0x"');
   }
+  const walletCapabilities = useCapabilitiesSafe({ chain });
+
   const value = useMemo(() => {
     const onchainKitConfig = {
-      address: address ?? null,
+      address: address ?? null, // this can maybe be updated to account.address
```

Comment:
yeah we need to deprecate the `address` story here, but that will require a much bigger PR.

Reply:
Are we keeping this comment?
```
this can maybe be updated to account.address
```